portal/server/helpers/isSuperUser.js

```diff
@@ -0,0 +1,4 @@
+const isSuperUser = (user) =>
+  user && user.bank && user.bank.id === '*';
```

Comment:
can we do
`const isSuperUser = (user) =>  user?.bank?.id === '*'`
?

Reply:
I tried this and for some reason portal/BSS UI does not like the x?.y?. syntax

Reply:
does it throw an error? or is it an error coming up from eslint?

Reply:
Both 🙃